tests/test_early_stopping.py

```diff
@@ -253,8 +254,8 @@ def test_early_stopping(self):
         """Tests early stopping."""
         # Set automatic_memory_optimization to false during testing
         nations = Nations()
-        model: Model = TransE(triples_factory=nations.training, automatic_memory_optimization=False)
-        evaluator = RankBasedEvaluator()
+        model: Model = TransE(triples_factory=nations.training)
+        evaluator = RankBasedEvaluator(automatic_memory_optimization=False)
```

Comment:
somewhere in this test the training loop also needs to be set explicitly

Reply:
Done in 9a6a26f0086974c9aeb924a4d0187d424664d23b.